### PR TITLE
Add new dead letter Q and attribute it to default Q

### PR DESCRIPTION
**Pull request by galCohen88**
Add new dead letter Q and attribute it to default Q

To enable the dead letter, we need to change the timeout

### CODE REVIEW

**Comment by robertkraig** [comment]
It would be nice to get this in the official repo.

**Comment by galCohen88** [comment]
I don't mind, I need approval for merging it

**Comment by robertkraig** [comment]
You might want to update the Readme.md file reguarding the `#edit-elasticmq-configuration-file` section on configuring the dead letter queue.

**Comment by galCohen88** [comment]
Added a reference in README

File: README.md [hunk 1]
```diff
@@ -105,6 +105,9 @@ The default configuration provisions ElasticMQ with a initial queue of the same
 
 To make use of this queue, point your client to: `http://localhost:9324/queue/default`.
 
+#### Dead letter queue
+The default dead letter queue is called dlq, configured in opt/elasticmq.conf
```
Comment (robertkraig) [inline README.md:109]: Maybe make a reference to: https://github.com/softwaremill/elasticmq#automatically-creating-queues-on-startup

Comment (galCohen88) [reply README.md]: Done

**Review by roribio — APPROVED** [review]
Looks good to me 💯 

Thank you for your contribution @robertkraig and @galCohen88!

**Pull request merged**